packages/packages/site-settings/src/hooks/__tests__/use-action-props.test.tsx

```diff
@@ -0,0 +1,45 @@
+import useActionProps from '../use-action-props';
+import { renderHook } from '@testing-library/react-hooks';
+import { runCommand, useRouteStatus } from '@elementor/v1-adapters';
+
+jest.mock( '@elementor/v1-adapters', () => ( {
+	runCommand: jest.fn(),
+	useRouteStatus: jest.fn(),
+} ) );
+
+describe( '@elementor/site-settings - useActionProps', () => {
+	afterEach( () => {
+		jest.clearAllMocks();
+	} );
+
+	it.each( [
+		{ title: 'open', input: { isActive: false, isBlocked: false }, expectedRoute: 'panel/global/open' },
+		{ title: 'close', input: { isActive: true, isBlocked: false }, expectedRoute: 'panel/global/close' },
```

Comment:
maybe split to multiple lines? kinda hard to read

Reply:
Done